diff --git a/dist/plugin-B4MlD0Bd.js b/dist/plugin-B4MlD0Bd.js
index 7ddbbcf1630129b133dbf3e8621942478b5348b8..d73f971411782d1952391aefbf1bb5ed7c35b0dc 100644
--- a/dist/plugin-B4MlD0Bd.js
+++ b/dist/plugin-B4MlD0Bd.js
@@ -1,5 +1,4 @@
 import assert from "node:assert/strict";
-import { isCSSRequest, normalizePath } from "vite";
 import fs from "node:fs";
 import path from "node:path";
 import { fileURLToPath } from "node:url";
@@ -90,6 +89,13 @@ function slash(p) {
 	return p.replace(windowsSlashRE, "/");
 }
 const isWindows = typeof process !== "undefined" && process.platform === "win32";
+function normalizePath(id) {
+	return path.posix.normalize(isWindows ? slash(id) : id);
+}
+const CSS_LANGS_RE = /\.(css|less|sass|scss|styl|stylus|pcss|postcss|sss)(?:$|\?)/;
+function isCSSRequest(request) {
+	return CSS_LANGS_RE.test(request);
+}
 function injectQuery(url, queryToInject) {
 	const { file, postfix } = splitFileAndPostfix(url);
 	return `${isWindows ? slash(file) : file}?${queryToInject}${postfix[0] === "?" ? `&${postfix.slice(1)}` : postfix}`;
@@ -502,7 +508,8 @@ function collectAssetDepsInner(fileName, bundle) {
 	};
 }
 function patchViteClientPlugin() {
-	const viteClientPath = normalizePath(fileURLToPath(import.meta.resolve("vite/dist/client/client.mjs")));
+	let viteClientPath
+	try { viteClientPath = normalizePath(fileURLToPath(import.meta.resolve("vite/dist/client/client.mjs"))); } catch { return null; }
 	function endIndexOf(code, searchValue) {
 		const i = code.lastIndexOf(searchValue);
 		return i === -1 ? i : i + searchValue.length;
